# Question: How many first class people survived the Titanic?

To answer this question I will be using observational study because I wont be changing any of the data. I will just be finding how many first class people survived. I will also be performing retrospective study and collecting my data using census because I will have all the people from first class so my conclusion will be more exact. Some biases that might be produced from my studies design is my result because I don't actually have all the people from the titanic so my study will be more biased.

In [10]:
import pandas as pd

In [11]:
df = pd.read_csv("titanic.csv")

In [12]:
df[(df["Survived"] == 1) & (df["Pclass"] == 1)]

PassengerId  Survived  Pclass  \
1              2         1       1   
3              4         1       1   
11            12         1       1   
23            24         1       1   
31            32         1       1   
..           ...       ...     ...   
862          863         1       1   
871          872         1       1   
879          880         1       1   
887          888         1       1   
889          890         1       1   

                                                  Name     Sex   Age  SibSp  \
1    Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
3         Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
11                            Bonnell, Miss. Elizabeth  female  58.0      0   
23                        Sloper, Mr. William Thompson    male  28.0      0   
31      Spencer, Mrs. William Augustus (Marie Eugenie)  female   NaN      1   
..                                                 ...     ...   ...    ...   
862  Swift, Mrs. Frederick Joel (Margaret Welles Ba...  female  48.0      0   
871   Beckwith, Mrs. Richard Leonard (Sallie Monypeny)  female  47.0      1   
879      Potter, Mrs. Thomas Jr (Lily Alexenia Wilson)  female  56.0      0   
887                       Graham, Miss. Margaret Edith  female  19.0      0   
889                              Behr, Mr. Karl Howell    male  26.0      0   

     Parch    Ticket      Fare Cabin Embarked  
1        0  PC 17599   71.2833   C85        C  
3        0    113803   53.1000  C123        S  
11       0    113783   26.5500  C103        S  
23       0    113788   35.5000    A6        S  
31       0  PC 17569  146.5208   B78        C  
..     ...       ...       ...   ...      ...  
862      0     17466   25.9292   D17        S  
871      1     11751   52.5542   D35        S  
879      1     11767   83.1583   C50        C  
887      0    112053   30.0000   B42        S  
889      0    111369   30.0000  C148        C  

[136 rows x 12 columns]

136 first class passengers survived the Titanic according to my data and the study I did. my answer is biased because I didn't have all the people from the Titanic in my data.